lib/python/gladevcp/tooledit_gtk.glade

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface>
-  <requires lib="gtk+" version="2.16"/>
+  <requires lib="gtk+" version="2.18"/>
```

Comment:
When I would save the file in the glade editor it would state that there was an incompatibility with 2.16 and that 2.18 was required. I updated to remove the error. If we need to revert we can.